app/assets/src/components/layout/Accordion.jsx

```diff
@@ -8,7 +8,7 @@ class Accordion extends React.Component {
 
   onToggle = () => {
     this.setState({
-      open: !this.state.open,
+      open: this.state.wasToggled ? !this.state.open : !this.props.open,
```

Comment:
Good fix. @MarkAZhang also had an idea how to make the accordion logic clearer: https://github.com/chanzuckerberg/idseq-web/pull/2014#discussion_r265372527